drivers/windows/windows.go

```diff
@@ -148,6 +149,15 @@ func (d *driver) parseNetworkOptions(id string, genericOptions map[string]string
 			config.DNSSuffix = value
 		case DNSServers:
 			config.DNSServers = value
+		case MacPool:
+			config.MacPools = make([]hcsshim.MacPool, 0)
+			s := strings.Split(value, ",")
+			for i := 0; i < len(s)-1; i += 2 {
```

Comment:
Oh, ok, I was assuming this will always come with an even number of elements.
As you suggested, I think it is better to explicitly fail in case of a malformed request, odd number of elements.